.github/workflows/build-release.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build and release
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - created
+
+jobs:
+  build-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18.x
+      - name: Set package.json version
+        uses: menduz/oddish-action@master
+        with:
+          deterministic-snapshot: true
+          only-update-versions: true
+      - name: Install
+        run: npm ci --legacy-peer-deps
+        env:
+          HUSKY: 0
+      - name: Build
+        run: npm run build
+        env:
+          NODE_PATH: 'src'
+          NODE_OPTIONS: '--max-old-space-size=6144'
+      - name: Publish
+        uses: menduz/oddish-action@master
+        with:
+          cwd: './dist'
+          deterministic-snapshot: true
+          provenance: false
+          registry-url: 'https://registry.npmjs.org'
+          access: public
+          gitlab-token: ${{ secrets.GITLAB_CDN_DEPLOYER_TOKEN }}
+          gitlab-pipeline-url: ${{ secrets.GITLAB_CDN_DEPLOYER_URL }}
```

Comment:
Would you mind asking for a devops admin to add these secrets to the repository?

Reply:
Done!

Reply:
Secrets already added!